packages/amplify-graphql-transformer-interfaces/src/transformer-context/synth-parameters.ts

```diff
@@ -7,4 +7,18 @@ export type SynthParameters = {
   identityPoolId?: string;
   adminRoles?: string[];
   enableIamAccess?: boolean;
+  deploymentIdentifier?: DeploymentIdentifier;
 };
+
+export interface SandboxDeploymentIdentifier {
```

Comment:
branch and sandbox deployments both have a name and namespace:
https://github.com/aws-amplify/amplify-backend/blob/39175772301cd9db58fd07ff6ed3e2c0e20fb876/packages/plugin-types/API.md?plain=1#L64-L74

In the case of sandbox, the name is the value of `package.json#name` and the namespace is either the local username or the value of `--identifier` supplied to the CLI